src/turn.js

```diff
@@ -1,31 +1,64 @@
 class Turn {
-	constructor(players) {
-		this.players = players;
-		this.guess = [];
-		this.endTurn = false;
-	}
+  constructor(round) {
+    this.round = round;
+    this.player = round.getCurrentPlayer();
+    round.choosePuzzle();
+    this.puzzle = round.currentPuzzle[0];
+    this.wheel = round.wheelData;
+  }
 
-	spinWheel() {
+  spinWheel(consonant) {
+    // spin wheel on DOM, get wedge value
+    if (wheel.wedge === 'BANKRUPT') {
+      this.player.currentScore = 0;
+    } else if (wheel.wedge === 'LOSE A TURN') {
+      this.endTurn();
+    } else {
+      this.player.currentScore += wheel.wedge;
+      this.guessConsonant(consonant);
```

Comment:
The invocation of `guessConsonant` is odd to me here. Is the flow that a user is clicking on a consonant in the DOM, and then that spins the wheel? I would expect a `spinWheel` method to be called upon a button click...like on a button called "SPIN WHEEL".

I would expect the `spinWheel` method to simply store the wheel value somewhere for reference later. Then a user would click on a consonant to invoke `guessConsonant(letter, wedgeValue)`. The `guessConsonant` method would then verify whether or not that letter was in the puzzle, and update the player's score accordingly.